interface/super/edit_globals.php

```diff
@@ -573,6 +573,35 @@ function validate_file(){
         "<input type='button' value='Default' onclick=\"document.forms[0].form_$i.color.fromString('" . attr($flddef) . "')\">\n";
     }
 
+    else if ($fldtype == 'visit_category') {
+        $sql = "SELECT pc_catid, pc_catname, pc_cattype 
+                FROM openemr_postcalendar_categories
+                WHERE pc_active = 1 ORDER BY pc_seq";
+        $result = sqlStatement($sql);
+        echo "<select name=\"form_{$i}\" id=\"form_{$i}\">\n";
+        echo "<option value='_blank'>" . xl('None') . "</option>";
+        while ($row = sqlFetchArray($result)) {
+            $id = $row['pc_catid'];
+            $name = $row['pc_catname'];
+            if ($id < 9 && $id != "5") {
+                continue;
+            }
+
+            if ($row['pc_cattype'] == 3 && !$GLOBALS['enable_group_therapy']) {
```

Comment:
To avoid confusion, would never show the group therapy categories. Group visits and normal visits are completed different things. So, would always ignore them no matter what the enable_group_therapy global is. If wanted to add a default_group_visit_category, then this would be a separate global.